src/components/BrandConfigProvider/theme/ThemeProvider.tsx

```diff
@@ -11,7 +12,7 @@ import { RecursivePartial } from '@styles';
 
 const defaultTheme = merge({}, defaultPaperTheme, baseTheme);
 
-export type Theme = typeof defaultTheme;
+export type Theme = typeof baseTheme & MD3Theme;
```

Comment:
This helps intellisense to show a simpler preview of the type of Theme.

Before:
<img width="932" alt="image" src="https://user-images.githubusercontent.com/2295908/224421265-58392d26-9682-4bb9-aa32-c6386320f863.png">


After:
<img width="629" alt="image" src="https://user-images.githubusercontent.com/2295908/224421190-56ac4b94-6b2b-43c3-b493-f5553ac0b8d3.png">